### PR TITLE
Release/v0.4.39

### PR DESCRIPTION
**Pull request by raswaters**

### CODE REVIEW

**Comment by bhayden53** [comment]
I had this problem with black recently, I think they pushed an update that gets installed in our actions now which wants to reformat some files. We'll have to check the failing action for which files black wants to change, and run black locally with the newest version, then push those changes.

File: changelog.md [hunk 1]
```diff
@@ -1,5 +1,4 @@
-- default base docker image set to CALDP_20221010_CAL_final
-- default crds update to hst_1038.pmap
-- pull cert chain from ami where the image is built
-- significant unit test coverage additions
-- node install bugfix in ami-rotation userdata
+- default base docker image set to CALDP_drizcosstis_CAL_rc2
```
[the remark this answers is not in the document]
Comment (raswaters) [reply changelog.md:1]: Will do, same for caldp.

**Comment by raswaters** [comment]
Who will take care of the black changes? Are those required or optional?

**Comment by bhayden53** [comment]
I will deal with the black changes, should be done within an hour

**Review by bhayden53 — APPROVED** [review]

**Review by alphasentaurii — APPROVED** [review]
LGTM!

**Pull request merged**